python/llm/test/benchmark/gpu/chatglm2.py

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This is to test latest ipex version, model is chatglm2
+import torch
+import os
+import time
+from bigdl.llm.transformers import AutoModel
+from transformers import AutoTokenizer
+import intel_extension_for_pytorch as ipex
+import numpy as np
+from itertools import chain
+import pathlib
+import argparse
+import json
+from benchmark_util import BenchmarkWrapper
```

Comment:
Can we just use the script under dev/benchmark instead of duplicate the entire file?